packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -210,15 +235,21 @@ class Auth {
             } else if (this.refreshToken) {
                 authorizationMethod = '_refreshAccessToken'
             }
-            return this[authorizationMethod](credentials).catch((error) => {
-                const retryErrors = [INVALID_TOKEN, EXPIRED_TOKEN]
-                if (retries === 0 && retryErrors.includes(error.message)) {
-                    retries = 1 // we only retry once
-                    this._clearAuth()
-                    return startLoginFlow()
-                }
-                throw error
-            })
+            return this[authorizationMethod](credentials)
+                .then((result) => {
+                    // Uncomment the following line for phased launch
+                    // this._onClient && this.createOCAPISession()
+                    return result
```

Comment:
Do we want to consider wrapping this in a feature flag that is enabled via the PWA Kit config file?

I am okay with the current approach as well as long as we clearly document to users that they MUST uncomment the above line if they are setting up a phased launch.

Reply:
Since we have a bunch of other (seemingly smaller) code changes we need to make manually to PWA Kit before deploying to a hybrid site. I believe having it this way is rather simpler than implementing a feature flag. This would feel consistent with change required for switching between cookie storage and local storage.